test/src/main/java/org/apache/accumulo/test/VolumeIT.java

```diff
@@ -488,4 +515,60 @@ private void updateConfig(Consumer<PropertiesConfiguration> updater) throws Exce
       config.write(out);
     }
   }
+
+  // Go through each tablet file in metadata and split the files into two files
+  // by adding two new entries that covers half of the file. This will test that
+  // files with ranges work properly with volume replacement
+  private void addRangesToFiles(AccumuloClient client, String tableName) throws Exception {
```

Comment:
There was another PR I reviewed that added a static util method for doing something similar.  Could that be used here?

Reply:
I just merged in #3824 so I can do a little refactoring to re-use some of that